app/gui/src/project-view/components/SelectionDropdown.vue

```diff
@@ -14,6 +14,8 @@ const _props = defineProps<{
   title?: string | undefined
   labelButton?: boolean
   alwaysShowArrow?: boolean
+  isTextDropdown?: boolean
+  heading?: string | null
```

Comment:
Not-present (`undefined`) or `null` would have the same meaning--let's avoid a distinction without a difference.

```suggestion
  heading?: string | undefined
```